Gopkg.lock

```diff
@@ -15,7 +15,12 @@
 
 [[projects]]
   name = "github.com/Azure/go-autorest"
-  packages = ["autorest","autorest/adal","autorest/azure","autorest/date"]
+  packages = [
```

Comment:
What version of `dep` are you using? I'm not sure why your `packages` items are split by newlines. It's not a big deal, but it makes this diff a little harder to parse.

Reply:
I noticed that. I was using the devel version but should be using the right version now.

Reply:
But it still formats the Gopkg.lock differently. . .
```└─▪ dep version
dep:
 version     : v0.3.2
 build date  : 2017-10-19
 git hash    : 8ddfc8a
 go version  : go1.9
 go compiler : gc
 platform    : linux/amd64
``` 
Please let me know if anything looks suspicious